### PR TITLE
chore: Update imports for supertokens-website-script to use supertokens-web-js-script

### PR DESCRIPTION
**Pull request by nkshah2**
## Summary of change

Changes made for:
- Quick setup
- Serverless 
- Troubleshooting guides

## Related issues
- 

## Checklist
- [ ] Algolia search needs to be updated? (If there is a new sub docs project, then yes)
- [ ] Sitemap needs to be updated? (If there is a new sub docs project, then yes)
- [ ] Checked for broken links? (Run `cd v2 && MODE=production npx docusaurus build`)
- [ ] Changes required to the demo apps corresponding to the docs?

## Remaining TODOs for this PR
- [ ]

### CODE REVIEW

File: v2/emailpassword/quick-setup/frontend.mdx [hunk 1]
```diff
@@ -138,10 +138,14 @@ SuperTokens.init({
...
+import supertokens from "supertokens-web-js-script";
 supertokens.init({
-    apiDomain: "^{form_apiDomain}",
-    apiBasePath: "^{form_apiBasePath}"
+    appInfo: {
+        apiDomain: "^{form_apiDomain}",
+        apiBasePath: "^{form_apiBasePath}",
+        appName: "...",
+    },
+    recipeList: [],
```
Comment (rishabhpoddar) [inline v2/emailpassword/quick-setup/frontend.mdx:148]: need to do session.init here?

File: v2/emailpassword/quick-setup/frontend.mdx [hunk 2]
```diff
@@ -138,10 +138,14 @@ SuperTokens.init({
 <AppInfoForm askForAPIDomain>
 
 ```tsx
-import supertokens from "supertokens-website-script";
+import supertokens from "supertokens-web-js-script";
```
Comment (rishabhpoddar) [inline v2/emailpassword/quick-setup/frontend.mdx:141]: Need to also change `<TabItem value="npm">` one

File: v2/emailpassword/serverless/with-netlify/frontend.mdx [hunk 3]
```diff
@@ -83,11 +83,18 @@ SuperTokens.init({
 >
 
 ```tsx
-import supertokens from "supertokens-website-script";
+import supertokens from "supertokens-web-js-script";
```
Comment (rishabhpoddar) [inline v2/emailpassword/serverless/with-netlify/frontend.mdx:86]: Need to also change <TabItem value="npm"> one

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: v2/emailpassword/quick-setup/frontend.mdx (hunk 1 above)
Comment (nkshah2) [reply v2/emailpassword/quick-setup/frontend.mdx:148]: Done in a different PR: https://github.com/supertokens/docs/pull/347/files#diff-a6fb346d682426623bbf3035d4196d1dae0d1189d6fffa8885d362e0dc817865

File: v2/emailpassword/serverless/with-netlify/frontend.mdx (hunk 3 above)
Comment (nkshah2) [reply v2/emailpassword/serverless/with-netlify/frontend.mdx:86]: Done in a separate PR

File: v2/emailpassword/quick-setup/frontend.mdx (hunk 2 above)
Comment (nkshah2) [reply v2/emailpassword/quick-setup/frontend.mdx:141]: Done in separate PR

File: v2/emailpassword/quick-setup/frontend.mdx (hunk 1 above)
Comment (rishabhpoddar) [inline v2/emailpassword/quick-setup/frontend.mdx:148]: need to call session init

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

**Review by rishabhpoddar — APPROVED** [review]

**Pull request merged**